manifests/certonly.pp

```diff
@@ -53,9 +53,9 @@
   }
 
   if ($custom_plugin) {
-    $command_start = "${letsencrypt_command} --text --agree-tos certonly "
+    $command_start = "${letsencrypt_command} --text --agree-tos -n certonly "
```

Comment:
I wonder if using "--non-interactive" instead of "-n" might be more self-documenting?